central/src/main/java/org/glowroot/central/repo/AggregateDaoImpl.java

```diff
@@ -2079,9 +2091,46 @@ private static String mergeRowIntoSummaries(Row row, Map<String, MutableSummary>
         }
         summary.totalDurationNanos += row.getDouble(i++);
         summary.transactionCount += row.getLong(i++);
+        summary.totalCpuNanos += row.getDouble(i++);
+        summary.totalAllocatedBytes += row.getDouble(i++);
         return transactionName;
     }
 
+    private static void bindAggregateForSummary(BoundStatement boundStatement, Aggregate aggregate,
+            int startIndex, TTL adjustedTTL) throws IOException {
+        int i = startIndex;
```

Comment:
Although sibling methods do not have this and it is a private method, a short Javadoc would be really nice:
When is an `IOException` thrown?
What are valid parameter ranges?
What does the method do (short summary)?